### PR TITLE
Temp: Unable to get this to compile...

### PR DESCRIPTION
**Pull request by x10an14**

### CODE REVIEW

**Comment by x10an14** [comment]
This is the error I get:
```
[2022-01-26 07:01:19] 0 x10an14@x10-desktop:~/.config/nixpkgs                                     (main $)
-> $ home-manager build --flake ~/.config/nixpkgs/
error: syntax error, unexpected '"', expecting '.' or '='

       at /nix/store/k5lhi4d91dvpbl4vwvx2m6jgf0snhmbv-source/nixFlakes.nix:2:28:

            1| { pkgs, ... }: {
            2|   pkgs.writeShellScriptBin "nixFlakes" ''
             |                            ^
            3|     exec ${pkgs.nixFlakes}/bin/nix --experimental-features "nix-command flakes" "$@"
(use '--show-trace' to show detailed location information)
[2022-01-26 07:25:53] 1 x10an14@x10-desktop:~/.config/nixpkgs           (nixFlakesUnstablePackagesSupport)
-> $
```

File: nixFlakes.nix [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{ pkgs, ... }: {
```
Comment (gvolpe) [inline nixFlakes.nix]: Remove the surrounding brackets and semicolon, should be like this

```nix
{ pkgs, ... }:

pkgs.writeShellScriptBin "nixFlakes" ''
  exec ${pkgs.nixFlakes}/bin/nix --experimental-features "nix-command flakes" "$@"
''
```

**Comment by x10an14** [comment]
@gvolpe Still no success =/
```
[2022-01-26 10:10:15] 0 x10an14@x10-desktop:~/.config/nixpkgs                                                                                                                                           (nixFlakesUnstablePackagesSupport)
-> $ home-manager switch --flake .
warning: Git tree '/home/x10an14/Documents/sourcehut/home-manager-dotfiles' is dirty
warning: Git tree '/home/x10an14/Documents/sourcehut/home-manager-dotfiles' is dirty
error: value is a set while a list was expected

       at /nix/store/k691msj3ah1idraykcrxmn7hs6idks6b-source/home.nix:12:16:

           11|   home = {
           12|     packages = myPackages ++ nixFlakes;
             |                ^
           13|   };
(use '--show-trace' to show detailed location information)
[2022-01-26 10:10:19] 1 x10an14@x10-desktop:~/.config/nixpkgs                                                                                                                                           (nixFlakesUnstablePackagesSupport)
-> $
```

**Comment by gvolpe** [comment]
The error message says it all 😉 

```
error: value is a set while a list was expected
```

Try `packages = myPackages ++ [ nixFlakes];`

**Pull request merged**